frontend/src/components/buttons/FormButton.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+export default function FormButton({ isSecondary, children, onClick }) {
+  FormButton.propTypes = {
+    isSecondary: PropTypes.bool,
+    children: PropTypes.string,
+    onClick: PropTypes.func,
+  }
+
+  return isSecondary ? (
+    <SecondaryButton type="button" onClick={onClick}>
+      {children}
+    </SecondaryButton>
+  ) : (
+    <PrimaryButton>{children}</PrimaryButton>
```

Comment:
I think the button names should be clearer. So that you can immediately see what these two buttons are used for.